docs/Using-Fleet/REST-API.md

```diff
@@ -3706,7 +3746,8 @@ List all configuration profiles for macOS hosts enrolled to Fleet's MDM that are
         "name": "Example profile",
         "identifier": "com.example.profile",
         "created_at": "2023-03-31T00:00:00Z",
-        "updated_at": "2023-03-31T00:00:00Z"
+        "updated_at": "2023-03-31T00:00:00Z",
+        "checksum": "dGVzdAo="
```

Comment:
This has extra spaces in the indentation as well; mind fixing this section while you're in here? 😇